gooey/gui/windows/base_window.py

```diff
@@ -158,7 +158,7 @@ def UpdateProgressBar(self, value):
       pb.Pulse()
     else:
       value = min(int(value), pb.GetRange())
-      if pb.GetValue() != value:
+      if pb.GetValue() != value or value == 0:

```

Comment:
bug fix: with no any progress bar updates `pb.GetValue()` returns 0, but progress bar in a pulse mode